EmployeeWebAPI/EmployeeWebAPI/Services/EmployeeService.cs

```diff
@@ -0,0 +1,169 @@
+﻿using EmployeeWebAPI.IServices;
+using EmployeeWebAPI.Model;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace EmployeeWebAPI.Services
+{
+    public class EmployeeService : IEmployeeService
+    {
+        private readonly TestDbContext1 _context1;
+        private readonly TestDbContext2 _context2;
+        private readonly TestDbContext3 _context3;
+
+        public EmployeeService(TestDbContext1 context1, TestDbContext2 context2, TestDbContext3 context3)
+        {
+            _context1 = context1;
+            _context2 = context2;
+            _context3 = context3;
+        }
+
+        public async Task<EmployeeModel> AddEmployee(EmployeeModel employee)
+        {
+            employee.Gender = GetGender(employee.Gender);
+            _context1.Employee.Add(employee);
+            await _context1.SaveChangesAsync();
+            employee.Employee_PK = 0;
+            _context2.Employee.Add(employee);
+            await _context2.SaveChangesAsync();
+            employee.Employee_PK = 0;
+            _context3.Employee.Add(employee);
+            await _context3.SaveChangesAsync();
+
+            return employee;
+        }
+
+        public string GetGender(string gender)
+        {
```

Comment:
Why Convert Enum To String and String To Enum, instead directly use the EnumGender in ViewModels and Db Models